.changeset/quiet-lies-share.md

```diff
@@ -0,0 +1,19 @@
+---
+"effect": major
```

Comment:
-> "patch"?

Reply:
It’s certainly a major change, but I agree just a patch version 😂

Reply:
that's a good point, if we are pre-release we should probably flag majors too